components/constants/index.js

```diff
@@ -1,5 +1,16 @@
 import { links } from './links'
 
+export const MybitTokenSaleAPIEndpoint = 'https://api.mybit.io';
+export const dayInSeconds = 86400;
+
+export const getSecondsUntilNextPeriod = (timestampStartTokenSale) => {
+  const currentDay = ((Math.floor(Date.now() / 1000) - timestampStartTokenSale) / dayInSeconds) + 1;
+  const past = currentDay % 1;
```

Comment:
This is the `modulus` or [modulo](http://www.java2s.com/Tutorial/JavaScript/0040__Operators/Modulus.htm) operator in JavaScript. You are dividing a `currentDay` by `1` and asking the remainder of it, which well, is always going to be `0`. Or am I missing something? 😅

Reply:
it doesn't work that way.

past = 0.5 if currentDay = 1.5.

Since I'm calculating the currentDay/Period off the timestamp we never really get an Int back. We need the remainder to calculate the remaining seconds until the next period. Well, there may be a different way to go about it as far as Math but that works.

Reply:
I see, didn't consider decimal values, carry on ☺️